docs/feature-roles.asciidoc

```diff
@@ -0,0 +1,444 @@
+[role="xpack"]
+[[feature-roles]]
+== Grant users access to secured resources
+
+You can use role-based access control to grant users access to secured
+resources. The roles that you set up depend on your organization's security
+requirements and the minimum privileges required to use specific features.
+
+Typically, you need to create the following separate roles:
+
+* <<privileges-to-setup-beats,Setup role>>: To set up index templates and
+other dependencies
+* <<privileges-to-publish-monitoring,Monitoring role>>: One for sending monitoring
+information, and another for viewing it
+* <<privileges-to-publish-events,Writer role>>: To publish events collected by {beatname_uc}.
+* <<privileges-agent-central-config,Central configuration management role>>: To view, create,
+update, and delete APM Agent central configurations.
```

Comment:
The described role can only read from APM Agent central configurations, since everything else was moved to Kibana.